package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openmrs/orderentry-owa",
-  "version": "1.2.3",
+  "version": "1.2.4-SNAPSHOT",
   "description": "OWA for placing an order, revising it, discontinuing it, and being able to look up a patient&#39;s active orders.",
```

Comment:
No idea if the 'SNAPSHOT' convention is frowned upon in the NPM world, but it is consistent with our Maven packaging and would seem to fit well with the idea of hosting in Maven Repo

Reply:
@ibacher  @brandones  @bmamlin  thoughts on this?

Reply:
I don't believe that versioning scheme will work with npm (`npm version` & `npm publish`) but probably that doesn't matter if you're not publishing using npm.

Reply:
Just to reinforce what @brandones said, `1.2.4-SNAPSHOT` is technically a valid SemVer, but in Maven land `-SNAPSHOT` functions more like a tag than a version identifier (so `1.2.4-SNAPSHOT` might resolve to `1.2.4-20210402.193656-5` or `1.2.4-20210402.191113-4` depending on when it's run). `npm` just doesn't use that convention, so it would want to distinguish between `1.2.4-SNAPSHOT.0` and `1.2.4-SNAPSHOT.1`. But assuming we're just using this internally and publishing to Maven (which will handle the `-SNAPSHOT` part), it's probably not that big of a deal.

Reply:
If there is a better place for us to publish both the latest build and all of our releases for a given owa, and for the latest builds to be available through a consistent tag/alias that doesn't require updating our distribution to depend on a new build number of every commit, I think we should consider it.  Given that this is exactly what Maven does, and we use Maven successfully this way with existing tooling for our omod and war artifacts, this seems like the best way to do this to me, at least for now.
We will need our SDK to be updated to support pulling OWAs out of Maven rather than (or in addition to) Bintray.

Reply:
I'm totally 100% in support of just using `-SNAPSHOT` in the version name for OWAs.